AdyenActions/Actions/VoucherAction.swift

```diff
@@ -105,9 +115,7 @@ internal protocol InstructionAwareVoucherAction {
 }
 
 /// Describes an action in which a voucher is presented to the shopper.
-public class GenericVoucherAction: Decodable,
-    AnyVoucherAction,
-    InstructionAwareVoucherAction {
+public class GenericVoucherAction: Decodable, AnyVoucherAction {
```

Comment:
why is this public? 🤔

Reply:
good point, maybe it shouldn't have been internal, but the damage is already done I am afraid :-).